### ISSUE TITLE
using Max31865 V2.0 without stepper-slot

### ISSUE THREAD

**Issue by trashypopashy** [bigtreetech/BIGTREETECH-MAX31865#4]
Hi everyone,
has someone connected the MAX31865 module from Bigtreetech to a Printerboard without using a stepper-slot?

**Comment by rh101** [comment]
Yes, it would be the same method as connecting V1.0 of the MAX31865 board.  Refer to the [V1.0 manual](https://github.com/bigtreetech/BIGTREETECH-MAX31865/blob/master/BIGTREETECH%20MAX31865%20V1.0/BIGTREETECH%20MAX31865%20V1.0%20Manual.pdf) on how you would do this.

Check page 2 of the [V2.0 manual](https://github.com/bigtreetech/BIGTREETECH-MAX31865/blob/master/BIGTREETECH%20MAX31865%20V2.0/BIGTREETECH%20MAX31865%20V2.0%20Manual.pdf) to get the correct pin-out.  Don't use the `BIGTREETECH MAX31865 V2.0-PIN.pdf` file for the pin-out, because it's not correct (doesn't show VIN etc).
